src/HybridSession.php

```diff
@@ -127,12 +108,15 @@ public function destroy($session_id)
         return true;
     }
 
-    #[\ReturnTypeWillChange]
-    public function gc($maxlifetime)
+    public function gc(int $maxlifetime): int|false
```

Comment:
```suggestion
    public function gc(int $maxlifetime): int
```
This never returns `false` - also is `false` even a valid return type for PHP 8.1? I thought that was introduced in 8.2

Reply:
Just tested this on PHP8.1. `int|false` is a valid return type ... `false` isn't.

I don't having a very specific return type here provides much value. The more generic return type gives us the flexibility to change our minds later on.